### ISSUE TITLE
distinction between “correct” and “user answered” pins

### ISSUE THREAD

**Issue by foxfabi** [foxfabi/h5p-geoquiz#1]
To match accessibility requirements, these two types of markers should be different in shape and not just color.

**Comment by foxfabi** [comment]
The correct answer pin is now different like proposed from jelena. Marco Luzi has created the needed png images.
